lib/src/se/llbit/chunky/ui/SliderAdjuster.java

```diff
@@ -73,6 +73,10 @@ public double getMax() {
     return max;
   }
 
+  public Slider getSlider() {
+    return valueSlider;
+  }
+
```

Comment:
The slider is an implementation detail of this component and shouldn't be exposed to the outside.